test/src/module/common/cryptoTest.c

```diff
@@ -329,9 +329,42 @@ testRun(void)
         TEST_RESULT_VOID(ioFilterFree(hash), "    free hash");
 
         // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("md5 hash - zero bytes");
+
         TEST_ASSIGN(hash, cryptoHashNew(strNew(HASH_TYPE_MD5)), "create md5 hash");
         TEST_RESULT_STR_Z(varStr(ioFilterResult(hash)), HASH_TYPE_MD5_ZERO, "    check empty hash");
 
+        // Exercise most of the conditions in the local MD5 code
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("md5 hash - mixed bytes");
+
+        TEST_ASSIGN(hash, cryptoHashNew(strNew(HASH_TYPE_MD5)), "create md5 hash");
+        TEST_RESULT_VOID(ioFilterProcessIn(hash, BUFSTRZ("1")), "    add 1");
```

Comment:
You have a TEST_TITLE for each test from line 332 to 367 so why are you adding spaces in the TEST_RESULT_* comments? These all "look" like new tests so we should treat the formatting as such.

Reply:
Fixed.